llmfoundry/callbacks/async_eval_callback.py

```diff
@@ -185,57 +193,116 @@ def validate_interval(interval: Union[str, int, Time],
     return result
 
 
+def validate_eval_run_config(
+        eval_run_config: Optional[Dict[str, Any]]) -> Dict[str, Any]:
+
+    if isinstance(eval_run_config, DictConfig):
```

Comment:
As with the other logger thing, I think you just want to be popping it with convert=True.